src/website/views/components/data-table/_examples.pug

```diff
@@ -4780,6 +4785,11 @@ h3 Portal -- Accordion Expanded -- Child Expanded
               <div class="chi-data-table__row-grand-child">
                 <div class="chi-data-table__cell -expandable"></div>
                 <div class="chi-data-table__cell -key -bold">
+                  <button class="chi-button -icon -flat -sm -expand" aria-label="Expand row">
```

Comment:
Looks like the code snippet is a bit off. I think the button is supposed to be on row-child level.

![Screenshot 2021-01-18 at 12 46 55](https://user-images.githubusercontent.com/25136131/104912770-ff647800-598c-11eb-9fa6-673e0d021630.png)

Reply:
@janibolkvadze good catch!